cron/monthly/verify-plans.js

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+/**
+ * Makes sure all collectives.plan are up-to-date, downgrading and cancelling plans if needed.
+ */
+
+// Only run on the first of the month
+const today = new Date();
+if (process.env.NODE_ENV === 'production' && today.getDate() !== 1) {
+  console.log('NODE_ENV is production and today is not the first of month, script aborted!');
+  process.exit();
+}
+
+process.env.PORT = 3066;
+
+import { Op } from 'sequelize';
+import { compact, findKey, get, groupBy, map, values, pipe } from 'lodash/fp';
+import debugLib from 'debug';
+
+import models from '../../server/models';
+import { promiseSeq } from '../../server/lib/utils';
+import plans, { PLANS_COLLECTIVE_SLUG } from '../../server/constants/plans';
+import orderStatus from '../../server/constants/order_status';
+import emailLib from '../../server/lib/email';
+
+const debug = debugLib('verify-plans');
+
+const d = new Date();
+d.setMonth(d.getMonth() - 1);
+
+const START_DATE = new Date(d.getFullYear(), d.getMonth(), 1);
+const END_DATE = new Date(d.getFullYear(), d.getMonth() + 1, 1);
+const REPORT_EMAIL = 'ops@opencollective.com';
+const BATCH_SIZE = 10;
+const EXISTING_PLANS_SLUGS = pipe(values, map(get('slug')), compact)(plans);
+
+const LEVELS = {
+  LEGACY: 'LEGACY',
+  DOWNGRADE: 'DOWNGRADE',
+  CANCEL: 'CANCEL',
+  EXCEPTION: 'EXCEPTION',
+};
+
+export async function run(options = {}) {
+  const collectives = await models.Collective.findAll({ where: { plan: { [Op.ne]: null } } });
+  debug(`There is/are ${collectives.length} subscribed to our plans...`);
+
+  const info = [];
+
+  await promiseSeq(
```

Comment:
I don't think the charge-subscription script is a good example to follow in best practices.

I would simplify that and just loop over collectives without concurrency.

```
for (const collective of collectives) {
 ... 
}
```